examples/jsm/libs/stats.module.js

```diff
@@ -139,16 +139,20 @@ Stats.Panel = function ( name, fg, bg ) {
 
 		dom: canvas,
 
+		getDisplayValue: round,
```

Comment:
I'm fine with making this change but we should clarify how to move on with cases when files in `jsm/libs` are modified. 

Normally, these files are just copied over and considered as read-only. If we modify them, they get out of sync compared to the original module. In this case: https://github.com/mrdoob/stats.js. The ideal workflow would be to modify the source in https://github.com/mrdoob/stats.js, make a release and then use the new build in `three.js`.

This workflow could be enforce by just deleting `examples/jsm/libs/stats.module.js` and import it directly from https://github.com/mrdoob/stats.js with a CDN.

Reply:
Sorry I forgot to mention that - I agree. @mrdoob what's your preference here?